lib/config.js

```diff
@@ -35,6 +36,7 @@ var path = require('path'),
 var config = {
     configFile: path.join(__dirname, '/../config', name + '.yml'),
     listenPort: 3000,
```

Comment:
because is a default value. You could go to line: https://github.com/telefonicaid/fiware-aiakos/pull/12/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R68 and see, how to configure from file